scripts/prepdocslib/searchmanager.py

```diff
@@ -57,14 +61,23 @@ def __init__(
         self.embeddings = embeddings
         self.search_images = search_images
 
-    async def create_index(self):
+    async def create_index(self, vectorizers: VectorSearchVectorizer = []):
         if self.search_info.verbose:
             print(f"Ensuring search index {self.search_info.index_name} exists")
 
         async with self.search_info.create_search_index_client() as search_index_client:
             fields = [
-                SimpleField(name="id", type="Edm.String", key=True),
+                SearchField(
+                    name="id",
+                    type="Edm.String",
+                    key=True,
+                    sortable=True,
```

Comment:
Adding an analyzer is allowed with downtime to the index. However - turning on attributes that were previously not enabled isn't backwards compatible. Can you add integrated vectorization in a way that doesn't break existing deployments?

Reply:
Done made it backward compatible based on the feature flag.